benchmarks/tensorexpr/benchmark.py

```diff
@@ -203,7 +203,7 @@ def dump_result(self, result_dict):
         if self.output_type == "json":
             print(json.dumps(result_dict))
         elif self.output_type == "stdout":
-            msg = "%s: %.2f us, SOL %.2f GB/s, algorithmic %.2f GB/s" % (
+            msg = "{}: {:.2f} us, SOL {:.2f} GB/s, algorithmic {:.2f} GB/s".format(
```

Comment:
Why not move it to f-string right away?

Reply:
Maybe the line will become too long so ruff decided not to. I can change

Reply:
I resulting line would be too long. Since this is an automated change I will create additional changes in separate PRs if needed.